frontend/src/scenes/onboarding/OnboardingAIConsent.tsx

```diff
@@ -27,7 +27,11 @@ export const OnboardingAIConsent = ({ stepKey }: { stepKey: OnboardingStepKey })
     const { updateOrganization } = useActions(organizationLogic)
 
     const isNotAdmin = useRestrictedArea({ minimumAccessLevel: OrganizationMembershipLevel.Admin })
-    const [aiEnabled, setAiEnabled] = useState(currentOrganization?.is_ai_data_processing_approved ?? false)
+
+    // New accounts have `default=True` on the database, but older accounts may have null/false
+    const [isEnabledByDefault] = useState(!!currentOrganization?.is_ai_data_processing_approved)
```

Comment:
Why is this useState, rather than just a plain const? There's certainly more overhead in the useState here than worth for the boolean computation

Reply:
Ah, the idea here was to be able to store the original value for the setting without the changes the user might've taken in this screen. I now see we actually don't store the changes to `organization`until we move to the next step, so this isn't needed